examples/example-rust/src/bin.rs

```diff
@@ -0,0 +1,47 @@
+// took from https://gist.github.com/mjohnsullivan/e5182707caf0a9dbdf2d
```

Comment:
Hey! Because this code was not hand-crafted by you, we did some snooping and found that the original gist adapted the code from Rosetta Code. The gist was not licensed, but Rosetta Code's was licensed under the GNU Free Documentation license. Would you mind adding a NOTICE file with the following?

> "bin.rs was copied from https://gist.github.com/mjohnsullivan/e5182707caf0a9dbdf2d, which was modified from Rosetta Code's example at http://rosettacode.org/wiki/Hello_world/Web_server#Rust. This is part of the GNU Emacs Lisp Reference Manual. bin.rs is available under the GNU Free Documentation License 1.2 unless otherwise noted."

Reply:
Actually, I saw this [from wikipedia](https://en.wikipedia.org/wiki/GNU_Free_Documentation_License) in reference to the original license:

> The GNU Free Documentation License (GNU FDL or simply GFDL) is a copyleft license for free documentation, designed by the Free Software Foundation (FSF) for the GNU Project. It is similar to the GNU General Public License, giving readers the rights to copy, redistribute, and modify (only when without "invariant sections" restrictions) a work and requires all copies and derivatives to be available under the same license.

This might need a review from our legal team since this project is MIT licensed unless you want to provide your own example. Let me know!

Reply:
Fixed :)